src/ray/ray_config.h

```diff
@@ -137,6 +145,8 @@ class RayConfig {
         node_manager_forward_task_retry_timeout_milliseconds_(1000),
         object_manager_pull_timeout_ms_(100),
         object_manager_push_timeout_ms_(10000),
+        object_manager_num_recent_pushes_(20),
```

Comment:
This is kind of a small number, and might not actually be high enough to IMPALA (50-100 actors per node, 100s of transfers per second). On the other hand increasing this is a bit scary due to the O(n^2) implications of using a deque.

Reply:
Actually could this metadata be stored inline in local_objects_?

For each local object, keep a `map<node, timestamp>` of last push.

Reply:
Great idea, I'm storing things in `local_objects_` now and got rid of `recent_pushes_`.